r/tests/testthat/test-altrep.R

```diff
@@ -54,6 +80,54 @@ test_that("altrep vectors from int32 and dbl arrays with no nulls", {
   expect_false(is_arrow_altrep(as.vector(v_dbl$Slice(1))))
 })
 
+test_that("element access methods for int32 ALTREP with no nulls", {
+  withr::local_options(list(arrow.use_altrep = TRUE))
+  original <- 1:1000
```

Comment:
Not that it matters much, but does this array need 1000 elements to test the behavior?